src/components/i18n-helpers.tsx

```diff
@@ -30,3 +32,27 @@ export const useLocaleFormat = (dateFormat) => {
     }
   };
 };
+
+export const useTranslationParagraphs = (
+  ns?: string,
+  options?: UseTranslationOptions,
+) => {
+  const { t, ...useTranslationResponse } = useTranslation(ns, options);
+
+  const tWithParagraphs = (key) => {
+    const texts = t(key).split(/\n\s*\n/);
```

Comment:
Does this cover CR LF for Windows machines? Obviously not but I'm wondering how a Windows text would behave here?

https://en.wikipedia.org/wiki/Newline#Representation

Reply:
Cool thank you for sharing this article. I knew there are differences but have never read about them in detail. 

Do we need to differentiate in our case? The new lines (e.g. insert `\n`) are added manually to translation.json. Or do I miss something?

Reply:
> When a windows user adds a newline the OS will insert \r\n.

...

> Do we need to differentiate in our case? The new lines (e.g. insert \n) are added manually to translation.json. Or do I miss something?

I have to agree with @hofmann-felix here. JSON does not support multi-line strings, so all line breaks will be added manually and will thus always be in the form of `\n`.
If we wanted to provide a bullet-proof solution that works with all sorts of inputs we would need to include a solution for Windows' `\r\n` line breaks, but in this case - we are only working with our very own JSON files - I claim that handling `\n` is enough.

Reply:
Oh I totally missed that fact. That we put in actual `\n` characters. Sorry for that confusion and thanks for the clarification.